lib/src/widgets/flutter_deck_progress_indicator.dart

```diff
@@ -0,0 +1,114 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_deck/src/flutter_deck.dart';
+import 'package:flutter_deck/src/flutter_deck_router.dart';
+
+/// A linear progress indicator that shows the current progress of the
+/// presentation.
+///
+/// The progress is calculated by the number of total steps in the presentation
+/// and the current step.
+class FlutterDeckProgressIndicator extends StatefulWidget {
+  /// Creates a progress indicator with a solid color.
+  const FlutterDeckProgressIndicator.solid({Color? color})
```

Comment:
To unify this with the `gradient` constructor and use positional parameters:
```suggestion
  const FlutterDeckProgressIndicator.solid([Color? color])
```

Reply:
I actually changed both constructor to be named constructors

Reply:
Sure, that works as well.